Week-3/assignment-3.4/refactoredFile.js

```diff
@@ -0,0 +1,20 @@
+function createStack() {
+    let that = {};
+    // console.log(that, "that");
+    that.items = [];
+    return {
+        push(item) {
+            that.items.push(item);
```

Comment:
try implementing same using arrow functions too

Reply:
@technikhil314 I have changed the code to arrow function and modified the implementation